src/data_storage/metatype_storage.ts

```diff
@@ -172,6 +176,13 @@ export default class MetatypeStorage extends PostgresStorage{
         }
     }
 
+    private static searchStatement(containerID: string, name: string): QueryConfig {
+        return {
+            text: `SELECT * FROM metatypes WHERE container_id = $1 AND NOT archived AND name LIKE '%${name}%'`,
```

Comment:
I don't think this is something to worry about now, but the wildcard search at the start of the string, in the WHERE clause, can cause issues performance-wise. Just might be something to visit down the line.

Reply:
Good point. Are you comfortable if I make an issue stating as much instead of solving that particular problem now?

Reply:
See issue #4